openmc/waste.py

```diff
@@ -0,0 +1,279 @@
+            "Dy154": 1.0e3,
+            "Ho166_m1": 2.0e-1,
+            "Hf178_m1": 9.0e3,
+            "Hf182": 2.0e-1,
+            "Re186_m1": 2.0e1,
+            "Ir192_m1": 1.0e0,
+            "Pt193": 2.0e8,
+            "Hg194": 5.0e-1,
+            "Pb202": 6.0e-1,
+            "Pb210": 3.0e7,
+            "Bi207": 9.0e3,
+            "Bi208": 8.0e-2,
+            "Bi210_m1": 1.0e0,
+            "Po209": 3.0e3,
+            "Ra226": 1.0e-1,
+            "Ra228": 3.0e7,
+            "Ac227": 5.0e5,
+            "Th229": 2.0e0,
+            "Th230": 3.0e-1,
+            "Th232": 1.0e-1,
+            "Pa231": 7.0e-1,
+            "U232": 3.0e1,
+            "U233": 2.0e1,
+            "U234": 9.0e1,
+            "U235": 2.0e0,
+            "Np236": 1.0e0,
+            "Np237": 1.0e0,
+            "Pu238": 7.0e4,
+            "Pu239": 1.0e3,
+            "Pu240": 1.0e3,
+            "Pu241": 2.0e3,
+            "Pu242": 1.0e3,
+            "Pu244": 9.0e-1,
+            "Am241": 5.0e1,
+            "Am242_m1": 3.0e2,
+            "Am243": 2.0e0,
+            "Cm243": 6.0e2,
+            "Cm244": 5.0e5,
+            "Cm245": 5.0e0,
+            "Cm246": 8.0e2,
+            "Cm248": 8.0e2,
+        }
+
+    elif limits == 'NRC_long':
+        # Specific activity limits for long-lived radionuclides from Table 1 in
+        # 10 CFR 61.55 in Ci/m3.
+        limits = {
+            'C14': 8.0,
+            'Tc99': 3.0,
+            'I129': 0.08,
+        }
+        if metal:
+            limits['C14'] = 80.0
+            limits['Ni59'] = 220.0
+            limits['Nb94'] = 0.2
+
+        # Convert values in nCi/g to Ci/m3
+        factor = (1e6 * mat.get_mass_density()) / 1e9
+        limits.update({
+            'Pu241': 3500.0 * factor,
+            'Cm242': 20000.0 * factor,
+            'Np237': 100.0 * factor,
+            'Pu238': 100.0 * factor,
+            'Pu239': 100.0 * factor,
+            'Pu240': 100.0 * factor,
+            'Pu242': 100.0 * factor,
+            'Pu244': 100.0 * factor,
+            'Am241': 100.0 * factor,
+            'Am243': 100.0 * factor,
+            'Cm243': 100.0 * factor,
+            'Cm244': 100.0 * factor,
+            'Cm245': 100.0 * factor,
+            'Cm246': 100.0 * factor,
+            'Cm247': 100.0 * factor,
+            'Cm248': 100.0 * factor,
+            'Bk247': 100.0 * factor,
+            'Cf249': 100.0 * factor,
+            'Cf250': 100.0 * factor,
+            'Cf251': 100.0 * factor,
+        })
+
+    elif limits == 'NRC_short_A':
+        # Get Class A specific activity limits for short-lived radionuclides
+        # from Table 2 in 10 CFR 61.55
+        limits = {
+            'H3': 40.0,
+            'Co60': 700.0,
+            'Ni63': 35.0 if metal else 3.5,
+            'Sr90': 0.04,
+            'Cs137': 1.0
+        }
+
+        # Add radionuclides with half-lives < 5 years to limits for class A
+        five_years = 60.0 * 60.0 * 24.0 * 365.25 * 5.0
```

Comment:
curious: Is there already a definition of seconds_per_year somewhere to rely on?

Reply:
We do, but it is within the `openmc.deplete` submodule (specifically, in `results.py`)  which is intentionally not imported by default.